modules/report/src/main/ReportApi.scala

```diff
@@ -161,6 +161,21 @@ final class ReportApi(
         case _ => funit
       }
 
+  def autoCheatDetectedReport(userId: User.ID, nbRecentCheatDetected: Fu[Int]): Funit =
```

Comment:
never take a future as an argument. Resolve the future before calling the function.

Don't do `f(futureInt)` but `futureInt flatMap f`